xclim/temperature.py

```diff
@@ -47,13 +47,13 @@ def cfprobe(self, da):
         checks.check_valid(da, 'standard_name', 'air_temperature')
 
 
-tg_mean = Tas(identifier='tg_mean',
-              units='K',
-              long_name="{freq} mean temperature",
-              standard_name="{freq} mean temperature",
-              description="{freq} of daily mean temperature.",
-              keywords='',
-              compute=_ind.tg_mean,)
+tmmean = Tas(identifier='tmmean',
+             units='K',
+             long_name="Mean daily mean temperature",
+             standard_name="air_temperature",
+             description="{freq} of daily mean temperature.",
```

Comment:
Will '{freq}' be translated to something more verbose using a dictionary?  Will need to apply both the freq and the method no? 'YS' yields annual but need to specify that used used .mean(dim='time') somehow so the description reads: 'Annual mean of daily mean temp' or equivalent

Reply:
Typo on my end

Reply:
Yes, see _attrs_mapping in utils.UnivariateIndicator.

Reply:
dacc